cmd/bosun/expr/funcs.go

```diff
@@ -244,6 +250,22 @@ func Duration(e *State, T miniprofiler.Timer, d string) (*Results, error) {
 	}, nil
 }
 
+func DropLe(e *State, T miniprofiler.Timer, series *Results, threshold float64) (*Results, error) {
+	for _, res := range series.Results {
+		nv := make(Series)
+		for k, v := range res.Value.Value().(Series) {
+			if float64(v) > threshold {
+				nv[k] = v
+			}
+		}
+		if len(nv) == 0 {
+			return nil, fmt.Errorf("drople: series %s is empty", res.Group)
```

Comment:
Ah, I think this is why I didn't want a drop function before. I'm not convinced it's a good idea to have a function that, with valid inputs, can produce an empty series. This needs more discussion.